woocommerce/wc-admin
# Dashboard: Rename class/feature to remove conflict with legacy dashboard widget

Fixes #2137 

This branch renames the `WC_Admin_Dashboard` to `WC_Admin_Analytics_Dashboard` to prevent the name collision that currently results in the legacy Woo Core dashboard widget from being displayed.

### Screenshots

![image](https://user-images.githubusercontent.com/22080/56982548-6e08fe80-6b36-11e9-85fb-8c334f701b2a.png)

### Detailed test instructions:

- On `master` of wc-admin installed/activated, note the bug by visiting the wp-admin dashboard and seeing that the WooCommerce Status widget is not shown ( see screen grab above )
- Install/npm start this branch, revisit the wp-admin dashboard and see that it is now shown
- Visit the WooCommerce Admin dashboard and verify it is still working
